### DISCUSSION TITLE
[BUG] Get-PnPTeamsTeam: Exception of type 'PnP.PowerShell.Commands.Model.Graph.GraphException' was thrown.

### DISCUSSION THREAD

**Discussion by YuriySamorodov** [pnp/powershell#2447] [General] [answered]
### Reporting an Issue or Missing Feature
I am reporting a bug

### Expected behavior 
`$team = Get-PnPTeamsTeam -Identity Team.Test` should not return any errors

### Actual behavior 
```
 Get-PnPTeamsTeam -Identity Team.GlobalInformationSecurity
Get-PnPTeamsTeam: Exception of type 'PnP.PowerShell.Commands.Model.Graph.GraphException' was thrown.
```
![image](https://user-images.githubusercontent.com/5260172/194904815-c35c7b06-adaf-420a-ba0e-28008fe020aa.png)

### Steps to reproduce behavior
```
Connect-PnPSite $adminUrl
Get-PnPTeamsTeam -Team Team
```
### What is the version of the Cmdlet module you are running?
```
Name    : PnP.PowerShell
Version : 1.11.99
```

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment, marked as answer]
Looks like it is missing necessary permissions.

Please ensure that the app you are using has the necessary permissions to retrieve teams data.

![image](https://user-images.githubusercontent.com/8832018/194939896-d765c154-b3fc-46f9-9c69-4e28aaaa75c4.png)

**Comment by YuriySamorodov** [comment]
Thanks @gautamdsheth . According to the docs https://pnp.github.io/powershell/cmdlets/Get-PnPTeamsTeam.html `Get-PnPTeamsTeam` requires the following permissions:
* Microsoft Graph API : One of Group.Read.All, Group.ReadWrite.All
